vms/avm/state/state.go

```diff
@@ -110,46 +114,76 @@ type State interface {
  *   |-- timestampKey -> timestamp
  *   '-- lastAcceptedKey -> lastAccepted
  */
-type state struct {
+type PreGState struct {
 	parser block.Parser
 	db     *versiondb.Database
 
 	modifiedUTXOs map[ids.ID]*avax.UTXO // map of modified UTXOID -> *UTXO if the UTXO is nil, it has been removed
-	utxoDB        database.Database
-	utxoState     avax.UTXOState
+	// TODO un-export databases post-migration
```

Comment:
Another idea is to just make the calling code use `NewWithFormat` instead of exporting these dbs